### DISCUSSION TITLE
What is the purpose of the 'context' input in hf.questionAnswering?

### DISCUSSION THREAD

**Discussion by ArsalaBangash** [huggingface/huggingface.js#263] [Q&A] [answered]
## Background

The args for QuestionAnswering are context and question.

```ts
export type QuestionAnsweringArgs = BaseArgs & {
	inputs: {
		context: string;
		question: string;
	};
};
```

The example in the docs is 

```ts
await hf.questionAnswering({
  model: 'deepset/roberta-base-squad2',
  inputs: {
    question: 'What is the capital of France?',
    context: 'The capital of France is Paris.'
  }
})
```

## Question

What is the purpose of the context input? Could the model return the right answer without it? 

Thank you!

**Comment by julien-c** [comment, marked as answer]
This is "classical" question answering, so the output is always extracted from the context (a span of text taken from the context)

Play with the widget here https://huggingface.co/deepset/roberta-base-squad2 to check it out

**Reply by ArsalaBangash** [reply]
I see. That clears it up for me.

Would a PR explaining such in the docs be welcome?
